### ISSUE TITLE
bank transfer not showing

### ISSUE THREAD

**Issue by martijnhovinga** [magento/magento2#2812]
Using Magento 2.0
When checking out for a product with a low price, only PayPal appears. When add another product and total order amount much higher, will have to go through the normal sequence to populate shipping details and also bank payment will appear.
Have set order amount to allow for bank payment in range $1 - 200,000
Suspected bank payment did not show up, since order amount was below threshold for free shipping. Then lowered free shipping below order amount. Yet bank payment did not show up. Wrong suspicion!

**Comment by ishakhsuvarov** [comment]
@martijnhovinga Thank you for reporting.
We have tried to reproduce the described issue but could not. Please provide detailed steps to reproduce (with the values you are using) and your store configuration.
Also check that `Payment from Applicable Countries` in Bank Transfer Settings has a correct value for your case.

**Comment by martijnhovinga** [comment]
Thanks for your quick reply. I checked again and you are right. The bank
transfer was there but I didn't see it since I had to scroll down to
make it visible.
Why is it that for the small order, the payment options appear right
away on my screen ('review & payments' in one shot) and for the bigger
amount only AFTER I fill out my 'shipping' details (see attached).

On another note, when I click 'go to checkout' in my other 2 websites
(babyplus.com.my and contoursbaby.com.my), with the same host and
running on the same Magento installation, the system hangs, see
attached. Or is that a configuration issue?

Many thanks and regards,
Martijn
